lambda_src/lambda_function.py

```diff
@@ -161,7 +161,10 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
             'headers': {'Content-Encoding': 'gzip'},
         }
 
-    response_length = len(dumps(response))
+    response_length = len(dumps(response).encode())
```

Comment:
Made this change so that this [line](https://github.com/Snowflake-Labs/geff/pull/67/files#diff-2b9d704e4645a913e70521e3fa194921702cf9e9aee4ce6573403adcd80c2d58R178) actually gets the bytes, instead of the string.